### PR TITLE
Steps to correct unable to schedule appointments

### PR DESCRIPTION
**Pull request by solemabrothers**

### CODE REVIEW

File: data_entry/art_card.md [hunk 1]
```diff
@@ -89,6 +89,23 @@ The default configuration restricts the maxiumum number of patients who can be s
 
 5. Click the "Save" button in the bottom right hand corner 
 
+###Determine the number of patients scheduled on a perticular.
```
Comment (ssmusoke) [inline data_entry/art_card.md]: @solemabrothers 

1. Add a screenshot with the error 

2. Change the title to "Unable to determine the number of patients scheduled due to error on the ART Clinical Assessment form"

File: data_entry/art_card.md [hunk 2]
```diff
@@ -89,6 +89,23 @@ The default configuration restricts the maxiumum number of patients who can be s
...
+This is because the user doesnt have the neccessary previllages. 
+Follow the  below here https://mets-programme.gitbook.io/ugandaemr-documentation/user_account_management/add_a_new_role_to_a_user_account and add the following roles
+
+1. Application: Edits Existing Encounters
+2. Application: Schedules Appointments
+3. Application: Schedules And Overbooks Appointments
+4. Data Entry
+5. System Developer
+6. Provider
+
```
Comment (ssmusoke) [inline data_entry/art_card.md]: @solemabrothers Remove these extra blank lines

File: data_entry/art_card.md [hunk 3]
```diff
@@ -89,6 +89,23 @@ The default configuration restricts the maxiumum number of patients who can be s
 
 5. Click the "Save" button in the bottom right hand corner 
 
+###Determine the number of patients scheduled on a perticular.
+
+On data entry, the user may get an error `unable to determine the number of patients scheduled due to error`.
+This is because the user doesnt have the neccessary previllages. 
+Follow the  below here https://mets-programme.gitbook.io/ugandaemr-documentation/user_account_management/add_a_new_role_to_a_user_account and add the following roles
```
Comment (ssmusoke) [inline data_entry/art_card.md]: @solemabrothers The link should be an internal document link

File: data_entry/art_card.md (hunk 1 above)
Comment (solemabrothers) [reply data_entry/art_card.md]: Done

File: data_entry/art_card.md [hunk 4]
```diff
@@ -89,6 +89,18 @@ The default configuration restricts the maxiumum number of patients who can be s
 
 5. Click the "Save" button in the bottom right hand corner 
 
+### Unable to determine the number of patients scheduled due to error on the ART Clinical Assessment form
```
Comment (ssmusoke) [inline data_entry/art_card.md]: @solemabrothers move to Common Errors section

**Pull request merged**